cmd/kro/commands/generate/instance.go

```diff
@@ -57,8 +59,21 @@ func generateInstance(rgd *v1alpha1.ResourceGraphDefinition) error {
 	if err != nil {
 		return fmt.Errorf("failed to create resource graph definition: %w", err)
 	}
+	em := emulator.NewEmulator()
+
+	crd := rgdGraph.Instance.GetCRD()
+	gvk := rgdGraph.Instance.GetCRD().GroupVersionKind()
+
+	s, err := schema.ConvertJSONSchemaPropsToSpecSchema(crd.Spec.Versions[0].Schema.OpenAPIV3Schema)
```

Comment:
I think this schema conversion has to happen exactly once for any given crd version so maybe its worth caching it. worth a todo?